ui/projects/etl-server/src/lib/pipelines/pipeline-list/pipeline-list.component.ts

```diff
@@ -49,11 +51,22 @@ export class PipelineListComponent implements OnInit {
           pipeline.display = pipeline.name;
         }
     
-        this.pipelineSections = this.processSections(pipelines, 0);
+        this.pipelines = pipelines;
+        this.pipelines.sort((a,b) => a[this.sortByField] > b[this.sortByField] ? 1 : -1);
+        this.pipelineSections = this.processSections(this.pipelines, 0);
       }
     });
     this.api.queryPipelines();
   }
+  get sortBySelected() {
+    return this.sortByField;
+  }
+
+  set sortBySelected(value) {
+    this.sortByField = value;
+    this.pipelines.sort((a,b) => a[this.sortByField] > b[this.sortByField] ? 1 : -1);
```

Comment:
Just thought of something - when sorting by update date, we want the one most recently updated to come first (like in gmail).
When sorting by name, we want an A-Z sorting.
From my reading of the code it's currently not like that, correct?